docs/src/components/propsTable/PropsTableTab.tsx

```diff
@@ -0,0 +1,82 @@
+import { Heading, Link } from '@aws-amplify/ui-react';
+import {
+  PropsTableData,
+  PropsTableSubComponentData,
+} from '../../../scripts/types/catalog';
+import { PropsTable } from './PropsTable';
+import { PropsTableExpander } from './PropsTableExpander';
+
+function PropsTableSet({
+  componentName,
+  componentPropsData,
+}: {
+  componentName: string;
+  componentPropsData: PropsTableSubComponentData;
+}) {
+  return (
+    <>
+      <Heading level={4} id={componentName}>
```

Comment:
```suggestion
      <Heading level={2} id={componentName} className="amplify-heading--4">
```
Make sure the heading levels follow the proper nesting; we can use classes to make them smaller.